README.md

```diff
@@ -188,60 +188,61 @@ data class Responses(
 
 This section documents the available CLI parameters for controlling what gets generated. This documentation is generated using: `./gradlew printCodeGenUsage`
 
```

Comment:
See line above, please regenerate this from the printCodeGenUsage command. It's not as pretty a table, but keepa code and instructions somewhat in sync

Reply:
sorry about that, missed that instruction there. i've fixed it.